src/clients/api/queries/getVTokenApySimulations/index.ts

```diff
@@ -16,8 +18,10 @@ const getVTokenApySimulations = async ({
   reserveFactorMantissa,
   interestRateModelContractAddress,
   isIsolatedPoolMarket,
+  asset,
 }: GetVTokenInterestRatesInput): Promise<GetVTokenApySimulationsOutput> => {
   const calls: ContractCallContext['calls'] = [];
+  const badDebtWei = new BigNumber(0).toFixed();
```

Comment:
@therealemjy should we add the `badDebt` value of an VToken/market? Maybe not for this PR, but in a followup

Reply:
We should yes. In this PR or a followup, it's up to you!

Reply:
In another one please, I want to briefly discuss it too

Reply:
Ok, sounds good to me